.github/workflows/continuous-integration-unit-php.yml

```
@@ -24,12 +24,25 @@ jobs:
     continue-on-error: ${{ matrix.experimental == true }}
     strategy:
       matrix:
-        php: ['7.4', '7.0', '5.6']
+        php: ['7.3', '7.0', '5.6']
         wp: ['latest']
+        multisite: [false]
+        coverage: [false]
         include:
           - php: '8.0'
             wp: 'trunk'
+            multisite: false
+            coverage: false
             experimental: true
+          - php: '7.4'
+            wp: 'latest'
+            multisite: true
+            coverage: false
```

```suggestion
            coverage: true
```
So just this @swissspidy

---

There is no need to run multisite with and without coverage right?

---

Fixed